source/whats-new.txt

```diff
@@ -33,6 +33,11 @@ Learn what's new in:
 What's New in 4.12
 ------------------
 
+.. important:: Upgrade to Version 4.12.1
```

Comment:
Issue: "upgrade to" in this context is somewhat ambiguous -- is this note talking about AN upgrade to the 4.12.1 or a recommendation that users upgrade TO 4.12.1. This is a nit, so feel free to leave it.  

```suggestion
.. important:: Upgrade Driver to Version 4.12.1
```